### PR TITLE
Lra 399 avm attention channel action cable is not needed

### PR DESCRIPTION
**Pull request by britaumich**
Removed all mentions of the attention_channel from the app.

To reload the dashboard added a js relod_dashboard script to the dashboard index page. We can remove this script if we don't want to reload the dashboard automatically.
The dashboard keeps reloading every 30 seconds. We can increase this time.

Added a refresh button to the dashboard.

To get rid of the attention channel from all Rails applications on your computer restart the computer.

### CODE REVIEW

File: app/javascript/controllers/dashboard_controller.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Controller } from "@hotwired/stimulus"
+export default class extends Controller {
+  static targets = ['refresh_button']
```
Comment (anantaUM) [inline app/javascript/controllers/dashboard_controller.js:3]: is this target getting used anywhere ?

File: app/javascript/controllers/dashboard_controller.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+  refresh() {
+    var url = window.location.pathname
+    Turbo.visit(url)
```
Comment (anantaUM) [inline app/javascript/controllers/dashboard_controller.js:7]: Here, we are loading the page. is there a way to just render the partial again ?

File: app/javascript/dashboard_reload.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+setTimeout(function () {
+    window.location.reload(true);
+}, 30000);
```
Comment (anantaUM) [inline app/javascript/dashboard_reload.js:3]: As discussed, I think this approach is not needed.

File: app/views/dashboard/index.html.erb [hunk 4]
```diff
@@ -1,14 +1,15 @@
...
-      <% @rooms.each do |room| %>
-        <li class="col-span-1 bg-white rounded-lg shadow-md divide-y divide-gray-400 px-2 grid place-items-stretch bg-white border border-gray-300 rounded-lg">
-          <%= render 'rooms/room_card', {room: room } %>
-        </li>
-      <% end %>
-    </ul>
+    <%= render 'listing' %>
   </div>
 
+  <script type="text/javascript" src="https://umlsait.atlassian.net/s/d41d8cd98f00b204e9800998ecf8427e-T/r5gghz/b/3/bc54840da492f9ca037209037ef0522a/_/download/batch/com.atlassian.jira.collector.plugin.jira-issue-collector-plugin:issuecollector/com.atlassian.jira.collector.plugin.jira-issue-collector-plugin:issuecollector.js?locale=en-US&collectorId=51ecac03"></script>
```
Comment (anantaUM) [inline app/views/dashboard/index.html.erb:13]: Why have we added jira collector script code here ?

File: app/views/dashboard/index.html.erb [hunk 5]
```diff
@@ -1,14 +1,15 @@
...
+  <%= javascript_include_tag 'dashboard_reload', async: true %>
```
Comment (anantaUM) [inline app/views/dashboard/index.html.erb:14]: As discussed, this approach is not needed.

**Review by anantaUM — COMMENTED** [review]
Added comments.

File: app/javascript/controllers/dashboard_controller.js (hunk 1 above)
Comment (britaumich) [reply app/javascript/controllers/dashboard_controller.js]: I removed dashboard_controller.js

File: app/javascript/controllers/dashboard_controller.js (hunk 2 above)
Comment (britaumich) [reply app/javascript/controllers/dashboard_controller.js]: I removed dashboard_controller.js

File: app/javascript/dashboard_reload.js (hunk 3 above)
Comment (britaumich) [reply app/javascript/dashboard_reload.js:3]: I removed the js file to automatically reload the dashboard.

File: app/views/dashboard/index.html.erb (hunk 4 above)
Comment (britaumich) [reply app/views/dashboard/index.html.erb]: Oops, mistake: copied too many lines.

File: app/views/dashboard/index.html.erb (hunk 5 above)
Comment (britaumich) [reply app/views/dashboard/index.html.erb]: Ok, I removed it.

**Comment by britaumich** [comment]
In the last commit, I propose a solution how to refresh only a frame with the rooms' listing:
[fa4469f](https://github.com/lsa-mis/avmonitor/pull/27/commits/fa4469fb792702fb866982e0109388012affa7fd)

File: app/views/dashboard/index.html.erb [hunk 6]
```diff
@@ -1,14 +1,11 @@
...
-      <% @rooms.each do |room| %>
-        <li class="col-span-1 bg-white rounded-lg shadow-md divide-y divide-gray-400 px-2 grid place-items-stretch bg-white border border-gray-300 rounded-lg">
-          <%= render 'rooms/room_card', {room: room } %>
-        </li>
-      <% end %>
-    </ul>
+  <div class="mb-2">
+    <%= form_tag('/dashboard') do %>
+      <%= submit_tag "Refresh", class: "px-2 rounded-md shadow-sm bg-gray-100 text-sm text-black text-center ring-1 ring-black ring-opacity-5 focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500 hover:bg-gray-400" %>
+      <div class="">
```
Comment (anantaUM) [inline app/views/dashboard/index.html.erb:6]: Consider removing the class attribute since currently there is no value assigned to it.

Comment (britaumich) [reply app/views/dashboard/index.html.erb]: Empty classes are removed: [aa98335](https://github.com/lsa-mis/avmonitor/pull/27/commits/aa9833555c9d4f4c08d69525fa7ff7f79d8f6c0d)

**Review by anantaUM — APPROVED** [review]

**Pull request merged**